electrum/gui/qt/main_window.py

```diff
@@ -2051,7 +2051,7 @@ def show_wallet_info(self):
         dialog.setMinimumSize(500, 100)
         mpk_list = self.wallet.get_master_public_keys()
         vbox = QVBoxLayout()
-        wallet_type = self.wallet.storage.get('wallet_type', '')
+        wallet_type = self.wallet.get_wallet_label()#self.wallet.storage.get('wallet_type', '')
```

Comment:
Is this comment necessary? If so, please specify a TODO with steps to remove it.

Reply:
Its not.